editor/src/components/editor/server.ts

```diff
@@ -257,13 +257,13 @@ export async function saveAsset(
   fileType: string,
   base64: string,
   imageId: string,
-): Promise<void> {
+): Promise<string | null> {
   try {
     return saveAssetRequest(projectId, fileType, base64, imageId)
   } catch (e) {
     // FIXME Client should show an error if server requests fail
     console.error(e)
-    return
+    return null
```

Comment:
is the `FIXME` here still up to date? as far as I know this function is either awaited or has a `.catch` at all call sites, so the try/catch might be removed from here (of course if this a proves to be a risky change it's not a big deal)